src/coreclr/scripts/cse_ml/jitml/constants.py

```diff
@@ -0,0 +1,50 @@
+"""Constants and parameters for the project."""
+
+from typing import Sequence
+
+import numpy as np
+from .method_context import MethodContext
+
+MIN_CSE = 3
+MAX_CSE = 16
```

Comment:
Is this a starting number for min and max CSEs and if this works, then we will extend further?

Reply:
That's correct.  This was the starting point to get something working.  We need to think through how to give the model the ability to see and select all CSEs (up to 64 which is the JIT's max).  Defining a new architecture is yet another project to work on.  I filed that as an issue here:  https://github.com/leculver/jitml/issues/8